src/schema/party/invite.ts

```diff
@@ -0,0 +1,21 @@
+import { Type } from "@sinclair/typebox";
+
+import { defineEndpoint } from "@/generator-helpers.js";
+import { userId } from "@/schema/definitions/userId";
+
+export default defineEndpoint({
+    source: "user",
+    target: "server",
+    description: "invite the target player to your current party",
+    request: {
+        data: Type.Object({
+            userId: Type.Ref(userId),
+        }),
+    },
+    response: [
+        { status: "success" },
+        { status: "failed", reason: "not_in_party" },
+        { status: "failed", reason: "invalid_user" },
+        { status: "failed", reason: "target_already_in_party" },
```

Comment:
Am I right in thinking `target_already_in_party` as a possible failed response here indicates it won't be possible to even send a party invite to someone if they're already in a party? Not sure I like that

Reply:
Yeah you’re right, I think this error should probably be for accepting an invitation when already in a party, will change.